PonyEdit.pro

```diff
@@ -61,11 +51,14 @@ macx {
     QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/lib/libssh2.1.dylib @executable_path/libssh2.1.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
 	QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/ssl/lib/libssl.1.0.0.dylib @executable_path/libssl.1.0.0.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
 	QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/ssl/lib/libcrypto.1.0.0.dylib @executable_path/libcrypto.1.0.0.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
+}
+
+linux|macx {
+    TARGET = PonyEdit
 
-	QMAKE_CFLAGS += -Werror -Wunused-parameter
-	QMAKE_CXXFLAGS += -Werror -Wunused-parameter
+    LIBS += -lz -lssh2 -lcrypto -lssl
 
-    LIBS	+= -lssh2 -lcrypto -lssl
+    QMAKE_CXXFLAGS += -Werror -Wunused-parameter -Weffc++ -include $$PWD/main/global.h
```

Comment:
I'm not wild about using the `-include` parameter here. It seems that Qt Creator doesn't parse it properly, causing the macros defined there to show as undefined where they're used.